code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/util/TimeUtil.kt

```diff
@@ -0,0 +1,20 @@
+/*
+  Copyright 2024 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.notificationbuilder.internal.util
+
+internal object TimeUtil {
+    /**
+     * Returns the current timestamp in seconds.
+     */
+    val currentTimestamp: Long
```

Comment:
Core already has a util method for this `TimeUtils.getUnixTimeInSeconds()`. Can we use that instead?